README.md

```diff
@@ -104,6 +105,7 @@ Note that:
 
 - **contrary to the defaults** of `rest-server`, the persistent data volume is located to `/data`.
 - By default, the image uses authentication.  To turn it off, set environment variable `DISABLE_AUTHENTICATION` to any value.
+- By default, the image loads the `.htpasswd` file from the persistent data volume (i.e. from `/data/.htpasswd`). To change the location of this file, set the environment variable `PASSWORD_FILE` to the path of the `.htpasswd` file.
```

Comment:
I think this might be unclear and confusing to users as it is now.

The value of the `PASSWORD_FILE` env var must be set to or considered in the context of the container. For example, if a user has their `.htpasswd` at `/some/different/path/.htpasswd`, then setting `PASSWORD_FILE` to this value will not make rest-server inside the container able to reach this file - instead of will fail with an error that it cannot touch that file. The reason is of course that given path doesn't exist *inside* the container.

Since I don't see much of a use case to set `PASSWORD_FILE` to a path other than something under `/data` unless one also bind-mounts or creates a volume at another path inside the container, we must inform the reader of this.

What about something like "Please note that the path you set here is used *inside* the container, which means that you need to make sure that your `.htpasswd` file is accessible at the give path, which is normally done by bind-mounting the file into a specific path in the container, or using a volume where the file is stored."?

Reply:
What about adding something like the following?

> Please note that this path must be accessible from inside the container and should be persisted. This is normally done by bind-mounting a path into the container or with another docker volume.

 The file doesn't need to exist prior to container creation since users could bind-mount a directory or attach a docker volume if they wanted to store the `.htpasswd` file separately from their data.

Reply:
That's fine by me. Sounds good, even though some people might still not understand what it means, but the main thing is that they understand the "inaccessible from inside the container", so by all means..